docker/Dockerfile.pdo-build

```diff
@@ -104,7 +104,7 @@ ENV PDO_INTERPRETER=${PDO_INTERPRETER}
 
 # - web-assembly/wasm/wawaka
 #   - Configure WAMR source
-ARG WAMR=WAMR-03-19-2020
+ARG WAMR=WAMR-03-30-2020
```

Comment:
Realize that only belatedly. But if we include now wamr as a submodule, we probably should also adapt the Dockerfile to use the submodule rather than relying on the WAMR build arg (which essentially would become obsolete)?

Reply:
The Dockerfile should already be using the submodule. I had left this WAMR build arg to ensure that we're checking out the right tag in the submodule. Like you said, that's probably redundant, at this point, though.